### PR TITLE
Force index usage

### PR DESCRIPTION
**Pull request by fvovan**
https://github.com/cargomedia/cm/issues/2309
Needed for `all=1` listing.
It completes the query to match "left prefix" rule for `{level:1, created:1}` index. (the index itself can not be reversed because we do `count` queries without sorting).

### CODE REVIEW

**Comment by fvovan** [comment]
@tomaszdurka please review

File: library/CM/Paging/Log.php [hunk 1]
```diff
@@ -119,6 +119,8 @@ protected function _getCriteria() {
 
         if (null !== $this->_filterLevelList) {
             $criteria['level'] = ['$in' => $this->_filterLevelList];
+        } else {
+            $criteria['level'] = ['$in' => array_values(CM_Log_Logger::getLevels())];
```
Comment (tomaszdurka) [inline library/CM/Paging/Log.php]: Let's rather make the whole param _filterLevelList not-nullable?
So in constructor we can set it to all levels by default.

```php
public function __construct(array $filterLevelList = null, $filterType = null, $aggregate = null, $ageMax = null) 
    if (null === $filterLevelList) {
        $filterLevelList = array_values(CM_Log_Logger::getLevels());
    } else {
        foreach ($filterLevelList as $level) {
            $level = (int) $level;
            if (!CM_Log_Logger::hasLevel($level)) {
                throw new CM_Exception_Invalid('Log level does not exist.', null, ['level' => $level]);
            }
        }
    }
```

**Comment by fvovan** [comment]
@tomaszdurka please review.
it think getter is not needed here, `_filterLevelList` is used only once

**Comment by fvovan** [comment]
it didn't help. will be reverted in https://github.com/cargomedia/cm/pull/2399/files

**Pull request merged**